### ISSUE TITLE
[ Chain id 124 / Network - decentralized web]

### ISSUE THREAD

**Issue by nlpvan-paul** [ethereum-lists/chains#314]
> Fill out the following to add your chain / network id, OR propose a Pull Request making your edits to the ```_data/chains.json``` file directly

```
{
  "name": "Decentralized Web Mainnet",
  "shortName": "dwu",
  "chain": "DWU",
  "network": "mainnet",
   "icon": "https://decentralized-web.tech/dwu.png",
 "chainId": 124,
  "networkId": 124,
  "rpc": ["https://decentralized-web.tech/blkch_tutor.htm"],
  "faucets": [],
  "infoURL: "https://decentralized-web.tech/dw_chain.php",
  "nativeCurrency": {"name":"Decentralized Web Utility","symbol":"DWU","decimals":18}
}

please add - chainid 124
if possible please contact us by e-mail to - connect@decentralized-web.tech 

```

**Comment by ligi** [comment]
the RPC looks invalid - also please submit as PR not issue

**Comment by nlpvan-paul** [comment]
Ok, I will update the rpc url to respond to ethereum post rpc and then provide it here

**Comment by nlpvan-paul** [comment]
Ok, I did update the rpc url to respond to ethereum post rpc , please check -

"rpc": ["https://decentralized-web.tech/dw_rpc.php"],

**Comment by nlpvan-paul** [comment]
Do I have to submit a new chain id request or wait for a response in this request?

**Comment by ligi** [comment]
RPC via PHP?
you need t make a PR

**Comment by nlpvan-paul** [comment]
The .php page responds to post requests in ethereum rpc api -
 json-rpc

| 
| 
|  | 
json-rpc


 |

 |

 |


It responds in the same way as for ex. -
https://dataseed1.binance.org:443
I am new to your group .Can you clarify what do you mean by PR?

    On Thursday, July 1, 2021, 9:02:57 PM PDT, ligi ***@***.***> wrote:  
 
 


RPC via PHP?
you need t make a PR

**Comment by nlpvan-paul** [comment]
Do I have to submit a new chain id request or wait for a response in this request?

**Comment by nlpvan-paul** [comment]
is this a robot response or a real person?

if a real person please do -
curl -X POST --data '{"jsonrpc":"2.0","method":"net_version","params":[],"id":67}' --url 'https://decentralized-web.tech/dw_rpc.php'

you can check other commands as well.

the .php  page responds to HTTP POST requests as defined in Ethereum json rpc api - https://eth.wiki/json-rpc/API

**Comment by ligi** [comment]
It is a real person - was just wondering - have never seen a node implemented in PHP and I have merged a lot of chains here.
Anyway it seems to work - so please submit a Pull-Request (PR)

**Comment by nlpvan-paul** [comment]
Thank you for confirming the functionality.

I am not very familiar or experienced in git updates - submitting a Pull-Request (PR)
In order not to mess up  your project if possible please do the PR for -
{
  "name": "Decentralized Web Mainnet",
  "shortName": "dwu",
  "chain": "DWU",
  "network": "mainnet",
   "icon": "https://decentralized-web.tech/dwu.png",
 "chainId": 124,
  "networkId": 124,
  "rpc": ["https://decentralized-web.tech/dw_rpc.php"],
  "faucets": [],
  "infoURL: "https://decentralized-web.tech/dw_chain.php",
  "nativeCurrency": {"name":"Decentralized Web Utility","symbol":"DWU","decimals":18}
}

If not possible - let me know and I can then go over and do the steps for PR.

**Comment by ligi** [comment]
please do the PR on your own - the CI will prevent a mess

**Comment by ligi** [comment]
closed by #319
